src/backend/funmooc/auth/templates/registration/login.html

```diff
@@ -0,0 +1,14 @@
+{% extends 'richie/base.html' %}
+
+{% block content %}
+  <h2>Login</h2>
+  <form method="post">
+    {% csrf_token %}
+    {{ form.as_p }}
+    <button type="submit">Login</button>
+  </form>
+  <br>
+  <p><strong>-- OR --</strong></p>
+  <a href="{% url 'social:begin' 'edx-oauth2' %}">Login with edX (oauth2)</a><br>
+  <a href="{% url 'social:begin' 'edx-oidc' %}">Login with edX (oidc)</a><br>
+{% endblock %}
```

Comment:
add i18n to this view?

But anyway we must bypass it for our login workflow right? I think I saw a setting for this in OpenedX

Reply:
AFAIU this login view cannot be bypassed, the only thing we can bypass _via_ OpenEdx LMS admin interface is the authorization agreement view for a registered app.

For now this template is a POC and it deserves more love (i18n, styles, etc.), but more important than that, we should agree on the login logic depending on the targetted LMS release, _i.e._ oauth2 for hawthorn and oidc for dogwood (+eucalyptus). We should discuss about this.